app/controllers/application_controller.rb

```diff
@@ -1174,6 +1174,7 @@ def get_sort_col
   def set_config(db_record) # pass in the db record, either @host or @vm
     @devices = []    # This will be an array of hashes to allow the rhtml to pull out each device field by name
     unless db_record.hardware.nil?
+      db_notes = db_record.hardware.annotation.nil? ? "<No notes have been entered for this VM>" : db_record.hardware.annotation
```

Comment:
Hasn't this line been removed in commit c54af0cdf1a69cc788b7f0e6c386d20f9d2f05ad ? (i.e. this has been 
incorrectly included during rebase).